### DISCUSSION TITLE
Could not connect with Geoserver if not on local Server

### DISCUSSION THREAD

**Discussion by DongwonGim** [kartoza/docker-geoserver#527] [Q&A]
I'm utilizing Openlayers in Next.js to render layers in Geoserver.
On the Next.js web on my local server, Geoserver is rendering the layers just fine, but on other PCs on the same network, the layers are not rendering. It's not a CORS error, it's just not connecting to Geoserver. How can I resolve this?

this is my remote PC error in same network 
<img width="1100" alt="스크린샷 2023-05-10 오후 6 29 24" src="https://github.com/kartoza/docker-geoserver/assets/18261851/80ee8ade-db64-4913-8489-8989f40710b6">

**Comment by NyakudyaA** [comment]
Are you able to access the default tomcat landing page?

**Reply by DongwonGim** [reply]
It is possible to access in my server computer but not possible in the others.
